tests/cli/formatters/ascii/TestAliasesFormatter.test_output.ref

```diff
@@ -0,0 +1,6 @@
+           ╷                              
+ [1m [0m[1mAlias   [0m[1m [0m│[1m [0m[1mDescription                [0m[1m [0m 
```

Comment:
An example of an expected reference file.
ANSI symbols are saved, `cat <file>` shows how it looks on the terminal.
Usually, there is no need to update the file manually but run `pytest --rich-gen`